joomla/joomla-cms
# [4.0] Improve add module tooltip

### Summary of Changes
Clarify the tooltip for Modules + icon is for Site Module.


### Testing Instructions
Log in to the administration
Move cursor over the Modules + icon to see tooltip.
